sqlx_v0.5.13.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/Cargo.toml b/Cargo.toml
```

Comment:
Should it be there? I guess not.

Reply:
apparently, dependency is available in crates.io again, removing the hack